src/main/java/uk/gov/hmcts/reform/amapi/models/FilterResource.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.reform.amapi.models;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import uk.gov.hmcts.reform.amlib.models.Resource;
+
+import java.util.Set;
+
+@Data
+@Builder
+@AllArgsConstructor
```

Comment:
Is that just needed for the testing service to deserialise request payload?

Reply:
yes